### DISCUSSION TITLE
Cesium build problem

### DISCUSSION THREAD

**Discussion by ZhuoHaoCoder** [pelicanmapping/osgearth#2558] [Support Q&A] [answered]
Hello, I am having trouble compiling OSGEarth and Cesium. In the CesiumIon.cpp file, the constructor of the Connection object called with the Connection object only has 3 parameters, while the latest version of Cesium_native's Connection constructor requires at least 4 parameters. Is there any way to solve this problem without lowering the version of Cesium_native? Please help me, thank you.

**Comment by jasonbeverage** [comment, marked as answer]
Not currently.  You need to use cesium native 0.31.   We have plans to support later versions of cesium native but nothing has been done yet on that front.

**Reply by ZhuoHaoCoder** [reply]
My problem has been solved. Thank you for your help
